class/person.rb

```diff
@@ -12,6 +12,7 @@ def initialize(age = nil, name = 'Unknown', parent_permission: true)
     @name = name
     @parent_permission = parent_permission
     @age = age
+    @rental = []
   end
 
```

Comment:
- [ ] As we are having a [many-to-many](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) relationship with rentals, it's necessary to add a method `add_rental` here too